app/views/reports/setup.js.erb

```diff
@@ -21,7 +21,6 @@ $('#defined_reports_step_2').html("<%= raw escape_javascript(render(:partial =>
 $('#report_selection').hide();
 $('#defined_reports, #defined_reports_step_2').show();
 // $('#defined_reports_step_1').dialog('close');
-$("select").selectpicker()
```

Comment:
@kyle-glick @wtholt So, fun times... I'm looking into it now, but it appears that the magic that selectpicker works on the html structure also totally borks the fancy javascript we have for the reporting module (checking dependencies, ajax loading of the next box, etc).

So turning the reporting dropdowns into the consistent selectpickers we use everywhere else, will possibly be difficult.